pam/go-loader/libpam-authd.pam-auth-update.in

```diff
@@ -0,0 +1,16 @@
+Name: Authd authentication
```

Comment:
See my comment on meson (and if so, we need to rely on shipping the right pam-auth-update file directly).